eoscostarica/guide.eoscostarica.io
# Add information on open source projects page

### Open source projects page

### What does this PR do?

- resolves #289 

### Steps to test
1. Go to "Developer Guidelines"
2. Open "Open Source Projects"
3. Check the content and the added projects: myvote eos dao, eden smart contracts, constancias municipales, eos starter token sale contract, and token sale platform

#### CheckList
